tests/acceptance/features/apiSpaces/changingFilesShare.feature

```diff
@@ -0,0 +1,97 @@
+@api @skipOnOcV10
+Feature:
+
+  Background:
+    Given using spaces DAV path
+    And these users have been created with default attributes and without skeleton files:
+      | username |
+      | Alice    |
+      | Brian    |
+
+  @issue-764
+  Scenario: Moving a file into a share as recipient
+    Given user "Alice" has created folder "/shared"
+    And user "Alice" has shared folder "/shared" with user "Brian"
+    And user "Brian" has accepted share "/shared" offered by user "Alice"
+    And user "Brian" has uploaded file with content "some data" to "/textfile0.txt"
+    When user "Brian" moves file "textfile0.txt" from space "Personal" to "shared/shared_file.txt" inside space "Shares Jail" using the WebDAV API
+    Then the HTTP status code should be "201"
+    And for user "Brian" folder "shared" of the space "Shares Jail" should contain these entries:
+      | shared_file.txt |
+    And for user "Alice" the space "Personal" should contain these entries:
+      | shared_file.txt |
```

Comment:
we can't perform the cross-space move detail discussion in this link: prevent the cross-space move. We have an existing scenario in  https://github.com/owncloud/ocis/blob/cb2822b8b741a67a1ac33500f7b98bb4f4b7afd2/tests/acceptance/features/apiSpaces/moveSpaces.feature#L131

Reply:
True, It's a duplicate. I'll remove this scenario from here.